src/renderer/pages/settings-page.component.tsx

```diff
@@ -137,6 +144,25 @@ export function SettingsPage() {
         i18nService.setLanguage(item.value);
     };
 
+    const handleVersionClick = async () => {
+        let isChangelogResolved = false;
+
+        await autoUpdater.showChangelog(await lastValueFrom(autoUpdater.getAppVersion()))
+        .then(() => {
+            setIsChangelogAvailable(() => true);
+        })
+        .catch(() => {
```

Comment:
Incorrect indentation for the then catch finally block; it should be indented one level deeper than the await to form a block with the await statement, rather than being an independent line of code.